RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -389,8 +389,10 @@ protected void executeRun(ExecutionContext context, boolean preventLoggingOnQuer
                         context.execute();
                         finallyStartJobsOfContext(context);
                     }
-                } finally {
+                } catch(Exception ex) {
+                    // (Maybe) abort jobs in strict mode
                     abortStartedJobsOfContext(context);
```

Comment:
If this is "(maybe)", why not call it `maybeAbortSTartedJobsOfContext(context)`?